glue/plugins/__init__.py

```diff
@@ -3,10 +3,17 @@
 from . import ginga_viewer
 
 
-def load_all_plugins():
+def register_plugins():
+    from ..config import qt_client, exporters
+    qt_client.add_plugin('glue.plugins.ginga_viewer')
+    exporters.add_plugin('glue.plugins.export_d3po')
+    exporters.add_plugin('glue.plugins.export_plotly')
```

Comment:
So is the idea that someone who has their own plugin would manually call the appropriate `registry.add_plugin` from their `config.py`?